common/src/cams/test-utilities/attorneys.mock.ts

```diff
@@ -1,94 +1,78 @@
-// TODO: Fakerize this list of trial attorneys.
-export const ATTORNEYS = [
+import { CamsRole } from '../roles';
```

Comment:
This is the source file we should probably export a handful of mock attorneys from for tests.